src/SprykerEco/Zed/Payone/Business/Payment/RequestSender/PayonePartialCaptureRequestSender.php

```diff
@@ -267,6 +268,27 @@ protected function getDeliveryCosts(OrderTransfer $orderTransfer): int
         return 0;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\PayonePartialOperationRequestTransfer $payonePartialOperationRequestTransfer
+     *
+     * @return bool
+     */
+    protected function checkNessesaryToAddDeliveryCosts(PayonePartialOperationRequestTransfer $payonePartialOperationRequestTransfer): bool
+    {
+        foreach ($payonePartialOperationRequestTransfer->getOrder()->getItems() as $itemTransfer) {
+            if (in_array($itemTransfer->getIdSalesOrderItem(), $payonePartialOperationRequestTransfer->getSalesOrderItemIds())) {
+                continue;
+            }
+            foreach ($itemTransfer->getStateHistory() as $itemState) {
+                if ($itemState->getName() === "shipped") {
```

Comment:
Please, move it to class const and use single quotes.

Reply:
Done.